pkg/cmd/cli/cmd/factory.go

```diff
@@ -5,14 +5,11 @@ import (
 
```

Comment:
@smarterclayton please take a look at this file - I'm missing something that I can't figure.